README.md

```diff
@@ -6,21 +6,6 @@ As of 2.0.0, algorithms exposed by ACCP are primarily backed by [AWS-LC](https:/
 
 [Security issue notifications](./CONTRIBUTING.md#security-issue-notifications)
 
-
-## Build Status
-Please be aware that both "Overkill" and "Dieharder" tests are known to be flakey.
-Both of these tests are flakey because they include entropy generation tests
-(specificaly, the [Dieharder tests](http://webhome.phy.duke.edu/~rgb/General/dieharder.php)).
-Entropy tests are unavoidably flakey because there is always a possibility that a high-quality
-random number generator will output data which looks non-random.
-(For example, even a fair coin will come up heads ten times in a row about one in a thousand trials.)
-
-| Build Name | master branch | develop branch |
-| ---------- | ------------- | -------------- |
-| x86        | ![Build Badge](https://codebuild.us-west-2.amazonaws.com/badges?uuid=eyJlbmNyeXB0ZWREYXRhIjoiRW4zZUhmeHlJbHRVQnNBZGZEbVJUa0pOK0J0MmtnNVB2dVZZSWhLbUtaNWYxNG96WWg4emN1SjJKL3VSUk9obFl0MnBtajBxejlVWDFiR3ppZGd3U1lrPSIsIml2UGFyYW1ldGVyU3BlYyI6IkFsUkpiMDRkRjZQb1U3Ly8iLCJtYXRlcmlhbFNldFNlcmlhbCI6MX0%3D&branch=master) | ![Build Badge](https://codebuild.us-west-2.amazonaws.com/badges?uuid=eyJlbmNyeXB0ZWREYXRhIjoiRW4zZUhmeHlJbHRVQnNBZGZEbVJUa0pOK0J0MmtnNVB2dVZZSWhLbUtaNWYxNG96WWg4emN1SjJKL3VSUk9obFl0MnBtajBxejlVWDFiR3ppZGd3U1lrPSIsIml2UGFyYW1ldGVyU3BlYyI6IkFsUkpiMDRkRjZQb1U3Ly8iLCJtYXRlcmlhbFNldFNlcmlhbCI6MX0%3D&branch=develop) |
```

Comment:
does the branch name change break these?

Reply:
Fixed it.